build.sh

```
@@ -5,41 +5,41 @@ export CI=false
 npm run build
 cd ..
 
-echo -e "\n***** Building New React App *****"
+echo -e "\n\033[1;37m\033[44m***** Building New React App *****\033[0m"
 cd app
 bash build.sh ${env}
 cd ..
 
-echo -e "\n***** Building browser extension *****"
+echo -e "\n\033[1;37m\033[44m***** Building browser extension *****\033[0m"
 cd browser-extension
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)